README.md

```diff
@@ -27,22 +27,64 @@ Multiple contact methods can be specified separated by ", " (without quotes).
 Requirements
 ------------
 
-The required Python packages are
+The required Python3 packages are
 
 * SQLAlchemy
 * sleekxmpp ≥1.0 (for the XMPP notification plugin)
-* TwitterAPI (for the Twitter notification plugin)
+* TwitterAPI twython (for the Twitter notification plugin)
 * [IRClib](https://bitbucket.org/jaraco/irc) (for the IRC notification plugin)
 
+On Debian systems install via apt with:
+
+    apt-get install python3-sqlalchemy python3-sleekxmpp python3-twython
+
+
 Setup
 -----
 
-Copy the `config.sample.py` to `config.py` and adept it to your needs. Then execute
+    git clone http://github.com/<this repositroy>/nodewatcher
+    cd nodewatcher
+    cp config.sample.py config.py
+
+and adept `config.py` to your needs.
+
+Then execute:
+
+    python3
+
 ```python
 from db import Base, engine
 Base.metadata.create_all(engine)
 ```
-to create the database structure. After that, all you need to do is call the `main.py`, for example via a cronjob in `/etc/cron.d/nodewatcher`:
+to create the database structure. 
+
+After that, all you need to do is call the `main.py`, for example via a cronjob in `/etc/cron.d/nodewatcher`:
 ```cron
-*/5 * * * * root /usr/local/src/nodewatcher/main.py
+*/5 * * * * root python3 /usr/local/src/nodewatcher/main.py
```

Comment:
Why call python3 explicitly and rely on a proper $PATH? If you want to be compatible with python3 in another location than `/usr/bin`, change the shebang to `/usr/bin/env python3`.